app/templates/components/crate-list.hbs

```diff
@@ -1,4 +1,4 @@
-<ul>
+<ol>
```

Comment:
Should [keyword-list](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/components/keyword-list.hbs) and [category-list](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/components/category-list.hbs) be changed to `ol`s too? They're both ordered by popularity, and those components are only used on the home page.

What about [crate-downloads-list](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/components/crate-downloads-list.hbs)? This is used on the currently-logged-in user's [dashboard](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/dashboard.hbs) and are ordered by recent downloads.

Not sure how far we should go with this in this PR-- there are lists of crates that are just `div`s rather than `ul`s right now that should also perhaps be `ol`s, like [every list of crates](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/crates.hbs#L68) (which should always have SOME order) or [the list of Latest Updates of crates you're following on your dashboard](https://github.com/rust-lang/crates.io/blob/e4f8898db18c6d5995812a92d9ef275a0fc525fb/app/templates/dashboard.hbs#L58)

Reply:
Yes, probably those need to be updated as well. Would rather do it in a follow-up PR if that's OK.

Reply:
Yup, that's totally fine!